templates/gettingstarted.html

```diff
@@ -19,41 +19,28 @@
             </p>
             <div class="command-line">
                 <div class="line comment"># create my-site instance</div>
-                <div class="line"><span class="prompt">$</span>mkvirtualenv my-site</div>
+                <div class="line"><span class="prompt">$</span>sudo apt-get install pipenv</div>
+                <div class="line"><span class="prompt">$</span>sudo apt-get install cookiecutter</div>
                 <div class="line"><span class="prompt">$</span>cookiecutter \</div>
-                <div class="line"><div class="tab">gh:inveniosoftware/cookiecutter-invenio-instance -c v3.0</div></div>
+                <div class="line"><div class="tab">gh:inveniosoftware/cookiecutter-invenio-instance -c v3.1</div></div>
                 <div class="line"><span class="prompt">$</span>cd my-site</div>
                 <div class="line comment"># start services (db, es, mq, etc)</div>
                 <div class="line"><span class="prompt">$</span>docker-compose up -d</div>
-                <div class="line comment"># bootstap my-site instance</div>
-                <div class="line"><span class="prompt">$</span>./scripts/bootstrap</div>
-            </div>
-            <p>
-               We follow by registering the record data model:
-            </p>
-            <div class="command-line">
-                <div class="line comment"># select instance</div>
-                <div class="line"><span class="prompt">$</span>workon my-site</div>
-                <div class="line comment"># scaffold data model in the same parent directory</div>
-                <div class="line"><span class="prompt">$</span>cookiecutter \</div>
-                <div class="line"><div class="tab">gh:inveniosoftware/cookiecutter-invenio-datamodel -c v3.0</div></div>
-                <div class="line comment"># install and register data model</div>
-                <div class="line"><span class="prompt">$</span>cd my-datamodel</div>
-                <div class="line"><span class="prompt">$</span>pip install -e .</div>
             </div>
             <p>
                We can now run the newly created instance:
             </p>
             <div class="command-line">
-                <div class="line comment"># select instance</div>
-                <div class="line"><span class="prompt">$</span>workon my-site</div>
+                <div class="line comment"># using pipenv</div>
                 <div class="line"><span class="prompt">$</span>cd my-site</div>
-                <div class="line comment"># setup database and indexes</div>
+                <div class="line"><span class="prompt">$</span>pipenv shell</div>
```

Comment:
This shouldn't be necessary to run any of the bootstrap/setup scripts, since they do `pipenv run ...` to run commands inside the virtualenv

Reply:
actually I wrote it in the way I've used it and tested it and I didn't use the pipenv run but the bootstrap, do you think it will be better to change it?

Reply:
Since it's less commands, I would remove it make a point of how "quick" our quickstart guide is

Reply:
done 🏎